### ISSUE TITLE
Tier 0.5 -Stone age: Vein search change suggestion

### ISSUE THREAD

**Issue by Bryfer** [GTNewHorizons/NewHorizons#1306]
Alot of players get locked in this phase for long periods of time, just searching for that last vein they havent found yet (chalcopyrite in my case)
Being stuck in this tier is really boring, and you cant progress any further before you have found all of these veins (If you are smart enough to start steam age before finding these veins, you'll just cry when you have to do all the machines again), but in reality you could if you wanted to go thru all steam age with only Malachite if you havent found any chalcopyrite yet. (you could also get ingots from chests if you get too bored vein searching)

My suggestion is to change these quests slightly 
![image](https://cloud.githubusercontent.com/assets/22825059/25102602/63ab10de-23b9-11e7-9851-8805be32c3ee.png)
Replace "Getting Iron, Copper, and Tin" quests with Smelt ingots quests (Similar to the current Tin quest)
and move the vein finding parts into optional quest instead. 

To make myself more understandable I made this image
![image](https://cloud.githubusercontent.com/assets/22825059/25103113/92f79874-23bb-11e7-99db-2e70dca1be7b.png)

What I suggest;
Rewrite Getting Iron to gather 72 Iron Ingots + new ingot icon
Rewrite Copper with Gather 24? Copper Ingots + new ingot icon
Keep Tin Quest as it is. (Gather 16 tin ingots) + new ingot icon
Make an quest for finding Chalcopyrite Vein
Make an quest for finding Magnetite Vein
Make an quest for finding Casserite Vein
Rename Malachite quest to Limonite vein (?)
Replace/rewrite Another fuel sorce into finding an Coal Vein
_there might be stone age veins I have forgotten here. If you got one in mind, mention it._

**Comment by Dream-Master** [comment]
Sure i can change it but i am not sure if this help realy. In the next tier you need plenty of Iron, Copper and Tin Ores. You get in this Stuck  problem only a bit later. And btw i play 10h real time in sp to get into bronze age and i play slow make all the quests.

**Comment by bigbass1997** [comment]
@Dream-Master I know your English is bad, but I really can not understand what you are saying.

I agree with Bryfer in that all vein finding quests should be replaced with Ingot requirements instead, say like 1 stack for each quest. (Iron, Copper, Tin, etc). If you still want the vein quests, fine, just put them all as optional, out of the way of the main quest line.

Aluminum Gravel Ore can remain the same as it is now, since this is not actually a vein and it can't be made into ingots.

This change would also prevent any further issues like this #1303 and every other issue where the different variants of ores prevent people from progressing.

In other words, why should someone be forced to find 2-3 different veins that contain copper or iron? There is no reason. To progress, the player needs the ingots, of these materials, not the ores, so requiring multiple kinds of ores is NOT helpful. Even singling it out to a single specific vein still does not solve the issues presented.

The Tin Quest already requires Ingots instead of ores; why Tin but not the others? Just make them all require ingots of each respective material, instead of a bunch of ore quests which don't even fully work with Better Questing.

**Comment by Dream-Master** [comment]
I need to make a rewrite then.

**Comment by hhheeexxx** [comment]
May be add a separate quest sheet about the search for veins of ores? With gifts for explore map. Tasks can be generated from gregs WorldGeneration.cfg. Need develop sheet composition and gift system. The rest can be done with a script,

**Comment by Dream-Master** [comment]
@hhheeexxx NO!

**Comment by Jayjayx5x1** [comment]
@hhheeexxx NO FREE GIFTS OR QUESTS! :P

**Comment by Dream-Master** [comment]
More quests are good but not 5 similar quests with 5 different ore type. If you have a better solution sure suggest it here. I am not against good and doable ideas. @hhheeexxx

**Comment by Dream-Master** [comment]
![image](https://cloud.githubusercontent.com/assets/2970688/25198811/eeb5aabe-2548-11e7-9091-16cccee00ee1.png)

**Comment by Jayjayx5x1** [comment]
@Dream-Master you could combine them and have one vein quest to balance it out if there are too many.

**Comment by Dream-Master** [comment]
@Jayjayx5x1 Why not the optional Quests. I like the idea from @Bryfer . Want to hit 1000 Quests soon :)

**Comment by hhheeexxx** [comment]
May be create ingame handbook about veins, but I don't now good and useful mod for that.
I think may be create quest sheet about veins with info about what world and what height for spawn. 
With simbolic gifts or no gifts for complite, like archevments. Or may be world basis quests lines.
May be start from discover world quest, for no info for no reachable worlds.
There is too many info for memory: in Overworld there is 22 different veins, in nether 11, in twilight 18, in end 13,  on galactic planets 6 - 16 per planet. 170 different ores spawn on in those dims (with Underdark there is 250+ ores). I think we need representative method for show this info.

**Comment by hhheeexxx** [comment]
May be create hidden quests. Like found ore - opens info about vein.

**Comment by Dream-Master** [comment]
Look at the GT Archivements Page most of the ores are covert up here. Or here https://docs.google.com/spreadsheets/d/1sRroNCV_4JpxrgKNivrOnSi-vz_-TX-a1-ll6WhwBJs/edit#gid=1348574161
